completions/fish/wl-paste.fish

```diff
@@ -0,0 +1,52 @@
+function __wayland_seats --description 'Print all wayland seats'
+    set -l seat
+
+    if type -q weston-info
+        set seat (weston-info 2>/dev/null | sed -n '/wl_seat/{n;s/\s*name: //;p}')
+    else if type -q loginctl
+        set seat (loginctl --no-legend --no-pager list-seats 2>/dev/null)
+    end
+
+    # Default seat if we have no other commands
+    if  test -z "$seat"
+        set seat "seat0"
+    end
+    
+    for i in $seat
+        echo "$i"
+    end
```

Comment:
Actually I do have yet another suggestion: here, you could do the same (get rid of the variable and the loop) if you move the `seat0` fallback to an `else` branch.

Don't forget that this function is in both source files.

Reply:
Added